website/src/components/APITable/index.tsx

```diff
@@ -1,27 +1,27 @@
-/**
```

Comment:
There's no reason to remove a copyright that is mandatory and required by our linter